wsmaster/che-core-api-factory-github/src/main/java/org/eclipse/che/api/factory/server/github/GithubUrl.java

```diff
@@ -174,7 +181,14 @@ public String location() {
    */
   public String rawFileLocation(String fileName) {
```

Comment:
Do we have anything else besides raw files?
https://docs.github.com/en/enterprise-server@3.4/admin/configuration/configuring-network-settings/enabling-subdomain-isolation

Reply:
No, we don't